### DISCUSSION TITLE
How to fit an image to a container

### DISCUSSION THREAD

**Discussion by Sterh20** [Budibase/budibase#12840] [Help] [answered]
I tried to use image's custom css `object-fit: cover;` and container's custom css `max-height: 210px;` but no luck
Currently I have this:
![image](https://github.com/Budibase/budibase/assets/102921356/98567801-5b6a-49ac-bd5a-c816129711b4)

I want to make the image look like this:
![image](https://github.com/Budibase/budibase/assets/102921356/5a547d10-3b0c-4b05-8bf3-127cf4f8d581)

What should I do? Is it possible at all?

**Comment by ConorWebb96** [comment, marked as answer]
Hey @Sterh20,

I've included the below images to demo the settings I used to get this look.
<img width="1512" alt="Screenshot 2024-01-23 at 07 45 52" src="https://github.com/Budibase/budibase/assets/126772285/72ffcede-972d-4227-9e46-41fb0dba9340">
<img width="1512" alt="Screenshot 2024-01-23 at 07 46 09" src="https://github.com/Budibase/budibase/assets/126772285/80f08392-6ba0-412d-afe7-af8a1efe5957">
<img width="1512" alt="Screenshot 2024-01-23 at 07 46 22" src="https://github.com/Budibase/budibase/assets/126772285/b0844bea-861e-42ac-b603-463db6f10a61">

Here is the end result.
<img width="1416" alt="Screenshot 2024-01-23 at 07 46 34" src="https://github.com/Budibase/budibase/assets/126772285/648d741a-9f46-4186-951e-248ffaba2bbf">

I hope this helps.

**Reply by Sterh20** [reply]
Thank you for your reply. My problem was solved by changing custom CSS of image to object-fit: cover; and setting height to fixed value in pixels. And I had to put out the image from it's container like you did. That was enough.
